notes.md

```diff
@@ -0,0 +1,12 @@
+TODO:
+
+* decide on synthdef submission method,
+  (eg. sent by clients? or already loaded on server?)
+* double check the SynthDescRelay
```

Comment:
I like the idea of clients adding synths.
In our session the other week, we were able to get the synthDescRelay working. The only caveat is that you have to have joined Utopia before any synthDefs are added in order to pick them up in your SynthDescLib, and they have to be in the SynthDescLib to be able to send messages to a remote instance.